frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -222,35 +222,35 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
   const metrics = [
     {
       key: 'usedMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes)',
+      query: 'node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes',
     },
     {
       key: 'totalMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes)',
+      query: 'node_memory_MemTotal_bytes',
     },
     {
       key: 'usedStorage',
-      query: 'sum by (instance) (node_filesystem_size_bytes - node_filesystem_free_bytes)',
+      query: 'instance:node_filesystem_usage:sum',
     },
     {
       key: 'totalStorage',
-      query: 'sum by (instance) (node_filesystem_size_bytes)',
+      query: 'node_filesystem_size_bytes{mountpoint="/"}',
     },
     {
       key: 'cpu',
-      query: 'sum by(instance) (instance:node_cpu:rate:sum)',
+      query: 'instance:node_cpu:rate:sum',
     },
     {
       key: 'pods',
-      query: 'sum by(node)(kubelet_running_pod_count)',
+      query: 'kubelet_running_pod_count',
     },
   ];
   const promises = metrics.map(({ key, query }) => {
     const url = getPrometheusURL({ endpoint: PrometheusEndpoint.QUERY, query });
     return coFetchJSON(url).then(({ data: { result } }) => {
       return result.reduce((acc, data) => {
         const value = Number(data.value[1]);
-        return _.set(acc, [key, data.metric.instance || data.metric.node], value);
+        return _.set(acc, [key, data.metric.node || data.metric.instance], value);
```

Comment:
after dropping the sum by on the pod count metric both the instance and node labels are there, but the instance label is the pod ip and not the name so it was setting the metric onto the wrong key, swapped these to just prefer node and then instance